WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -28,6 +28,7 @@
 
 import org.greenrobot.eventbus.Subscribe;
 import org.greenrobot.eventbus.ThreadMode;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Do we really need to use this Jetbrains annotation when we're already using `android.support.annotation.NonNull` in this fragment?